lib/rules/template-no-let-reference.js

```diff
@@ -0,0 +1,44 @@
+const ERROR_MESSAGE =
+  'update-able variables are not supported in templates, reference a const variable';
+
+/** @type {import('eslint').Rule.RuleModule} */
+module.exports = {
+  ERROR_MESSAGE,
```

Comment:
```suggestion
  ERROR_MESSAGE,
```

Remove since we're replacing it with messageIds: https://eslint.org/docs/latest/extend/custom-rules#messageids